pillar/heroku/ocw-studio.sls

```diff
@@ -66,9 +70,14 @@ heroku:
   config_vars:
     ALLOWED_HOSTS: '["*"]'
     AWS_ACCESS_KEY_ID:  __vault__:cache:aws-mitx/creds/ocw-studio-app-{{ env_data.env }}>data>access_key
+    AWS_PREVIEW_BUCKET_NAME: 'ocw-content-draft-{{ env_data.env }}'
+    AWS_PUBLISH_BUCKET_NAME: 'ocw-content-live-{{ env_data.env }}'
```

Comment:
I'm assuming `{{ env_data.env }}` == `qa` on RC and whatever it needs to be on production